modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -14,14 +14,17 @@ The Red Hat Telco Radio Access Network (RAN) version {product-version} solution
 |Product
 |Software version
 
+|Hub cluster {product-title} version
+|4.8 or 4.9
+
 |ZTP GitOps plugin
-|4.11
+|4.10
 
 |{rh-rhacm-first}
-|{rh-rhacm-version}
+|2.3 or 2.4
```

Comment:
@yliu127 can you check these versions. I think 4.10 was tested with ACM 2.4 and 2.5 and gitops 1.4?

Reply:
This is what we documented at time of release: https://docs.openshift.com/container-platform/4.10/scalability_and_performance/ztp-deploying-disconnected.html#ztp-preparing-the-hub-cluster-for-ztp_ztp-deploying-disconnected

Reply:
correct. CC @aireilly, 4.10 is tested with these versions: https://docs.google.com/spreadsheets/d/1ui8Jw9oE9-wCbmU-AqLkiLA4UXyYRoYhUZEt445Zaa4/edit?usp=sharing